### ISSUE TITLE
Failed to evaluate rule: unable to find context value: singleton

### ISSUE THREAD

**Issue by jmle** [konveyor/analyzer-lsp#226]
When running the following rule
```yaml
- customVariables: []
  effort: 1
  labels:
  - konveyor.io/target=eap7
  message: When a singleton EJB bean class implements `javax.ejb.SessionBean` interface,
    this interface should be removed from the implements clause.. All methods declared
    in `javax.ejb.SessionBean` interface (see below) that are implemented in the bean
    class or its super classes should be checked for `@Override` annotation and remove
    this annotation too if present.. Methods declared by `javax.ejb.SessionBean` interface:.
    ```. void setSessionContext(SessionContext ctx);. void ejbRemove();. void ejbActivate();.
    void ejbPassivate();. ```
  ruleID: singleton-sessionbean-00001
  when:
    or:
    - as: singleton
      java.referenced:
        location: ANNOTATION
        pattern: javax.ejb.Singleton
    - as: sessionbean
      from: singleton
      java.referenced:
        location: IMPLEMENTS_TYPE
        pattern: javax.ejb.SessionBean
```
against
```java
package org.example;

import javax.ejb.SessionBean;
import javax.ejb.Singleton;

@Singleton
public class Bean implements SessionBean {
}
```
the following error appears:
```
ERRO[0005] failed to evaluate rule                       error="unable to find context value: singleton" ruleID=singleton-sessionbean-00001
```

This might be related to the failure of [this test](https://docs.google.com/spreadsheets/d/19_dL8ZiBu0ULghVdpLsx0zBhF8DcGnjwMwslIcQytoU/edit#gid=372217138&range=A205) in the failing tests document.
